src/components/Toolbox.vue

```diff
@@ -1,64 +1,106 @@
 <template>
   <div>
-    <div id="toolbox-container">
-      <q-expansion-item
-        v-for="(category, title) in types"
-        :key="title"
-        :label="title"
-        header-style="background-color: #303030; border-bottom: solid #505050 3px"
-      >
-        <div class="q-py-sm">
-          <div class="node-list-element row justify-start q-mx-sm q-my-none"
-                v-for="(type, component) in category"
-                :key="type.spec.id"
-          >
-            <div class="col-1">
-              <div class="tree-line" />
-            </div>
-            <div class="col q-py-sm">
-              <q-btn
-                color="primary"
-                class="full-width row"
-                @click="$emit('add', component, type.spec)"
-              >
-                <img id="icon" class="non-selectable col-auto" v-if="type.spec.icon" :src="type.spec.icon" />
-                <div class="col">{{ type.spec.title }}</div>
-              </q-btn>
+    <div id="toolbox-wrapper" :style="'left:' + toolboxOffset + 'px;'">
+      <div id="toolbox-container" :style="'min-width: ' + toolboxWidth + 'px;'">
```

Comment:
These could be changed into template strings using back-ticks (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)). That's how we've usually done it so far.
Alternatively you could use a native object (``:style="{ left: `${toolboxOffset}px\`, ... }"``, which we've sometimes done as well.